docs/_data/_routes.yml

```diff
@@ -1,120 +1,98 @@
 home:
   href: home
-  title: Welcome to ExpensifyHelp!
-  description: Find answers by using the search bar in the left-hand menu or by clicking the version of Expensify you're using to access the relevant resources.
+  title: Need some help?
+  description: Explore the menu or search for a specific topic. 
 
 platforms:
   - href: expensify-classic
     title: Expensify Classic
     hub-title: Expensify Classic - Help & Resources
-    hub-description: Questions? Find the answers by clicking a Category or using the search bar located in the left-hand menu.
+    hub-description: Need some help? Explore the menu or search for a specific topic.
     url: www.expensify.com
-    description: "Your account settings look like this:"
     image: /assets/images/settings-old-dot.svg
   
     # Hubs are comprised of subcategories and articles. Subcategories contain multiple related articles, but there can be standalone articles as well
     hubs:
       - href: getting-started
         title: Getting Started
         icon: /assets/images/accounting.svg
-        description: From setting up your account to ensuring you get the most out of Expensify’s suite of features, click here to get started on streamlining your expense management journey.
 
       - href: account-settings
         title: Account Settings
         icon: /assets/images/gears.svg
-        description: Discover how to personalize your profile, add secondary logins, and grant delegated access to employees with our comprehensive guide on Account Settings.
 
       - href: bank-accounts-and-credit-cards
         title: Bank Accounts & Credit Cards
         icon: /assets/images/bank-card.svg
-        description: Find out how to connect Expensify to your financial institutions, track credit card transactions, and best practices for reconciling company cards.
 
       - href: billing-and-subscriptions
         title: Billing & Subscriptions
         icon: /assets/images/subscription-annual.svg
-        description: Here is where you can review Expensify's billing and subscription options, plan types, and payment methods.
```

Comment:
@ren-jones why have we removed the description for hubs?

<img width="1134" alt="image" src="https://github.com/Expensify/App/assets/29673073/85ea309e-26ae-4b91-8270-7ed02854a5c6">

Reply:
The descriptions added too much low-value text. These options are self explanatory without additional text muddying up the page, so I just removed them. I mentioned to Rushat that this may mean that we need to update the formatting of this page so that the icon and title better fit the squares. But this already looks much cleaner